src/app/forms/dynamic-form/handler/dynamic-form-functions.handler.ts

```diff
@@ -0,0 +1,31 @@
+import { FormGroup } from '@angular/forms';
+import { Observable } from 'rxjs/Observable';
+
+/**
+ * @author mfrank
+ */
+// ToDo art des interfaces definieren (z.b. handler), oder allgemein (interface) bleiben
+export interface DynamicFormFunctionsHandler<D>
+{
+    /**
+     * Called after valid form validation.
+     *
+     * @param formData
+     */
+    save:(formData:D) => void;
+
+    /**
+     * Called after valid form validation and when a REST URL is set.
+     *
+     * @param observable
+     */
+    saved:(observable:Observable<D>) => void;
```

Comment:
`save` und `saved` sind sehr sehr nahe beinander vom Naming.
Soll saved hier die Vergangenheitsform darstellen oder finden wir hierfür ein besseres Naming?

Reply:
save() gibt die values des form zurück, so das in der implementierenden Component das speichern übernommen werden muss.
saved() gibt ein observable zurück, nachdem der terra-dynamic-form.service das Forumular gespeichert hat.

Besseres Naming bietet sich vermutlich an, um Verwirrung zu vermeiden.